### ISSUE TITLE
About mutual info score

### ISSUE THREAD

**Issue by sk-g** [shubhangi-tandon/FashionMnistClassification#1]
Assuming this is still alive, wrt this line
```
get distributions for T, X and Y and calculate mutual info score using sklearn.mutual_info_score
```
I think you could use the KL divergence metric intermittently in between epochs. 
This is because we can express mutual information with marginal probabilities as their KL divergence. 
See [this](https://en.wikipedia.org/wiki/Mutual_information#Relation_to_Kullback%E2%80%93Leibler_divergence).
Can try something like this:
```
dist1 = K.variable()
dist2 = K.variable()
cxe = K.kullback_leibler_divergence(dist1,dist2).eval(session = K.get_session())

```
